rero_ils/modules/api.py

```diff
@@ -466,8 +479,9 @@ def delete(self, record):
         :param record: Record instance.
         """
         return_value = super().delete(record)
-        index_name, doc_type = current_record_to_index(record)
+        index_name, _ = current_record_to_index(record)
         current_search.flush_and_refresh(index_name)
+        self.after_delete_record(record)
```

Comment:
It can be nice to implement this a done for the invenio-records extensions.

Reply:
With `invenio-record-resources` it will be like this.